packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -943,6 +1009,15 @@ export class Vector3 {
         return Quaternion.RotationYawPitchRoll(this._y, this._x, this._z);
     }
 
+	/**
+	 * Converts the current Vector3 to its polar coordinates
+	 * @returns the current polar coordinates (r, theta, phi)
+	 */
+	public toPolar(): Vector3{
+		let angles = Vector3.GetRotationBetweenVectors(Vector3.Zero(), this);

```

Comment:
GetRotationBetweenVectors does not exist seems to be now called RotationBetweenVectors and as a method should be camel case